src/layer/tile/GridLayer.js

```diff
@@ -413,15 +413,19 @@ L.GridLayer = L.Layer.extend({
 		if (!map) { return; }
 
 		if (center === undefined) { center = map.getCenter(); }
-		if (zoom === undefined) { zoom = Math.round(map.getZoom()); }
+		if (zoom === undefined) { zoom = map.getZoom(); }
 
 		if (zoom > this.options.maxZoom ||
 			zoom < this.options.minZoom) { return; }
```

Comment:
I'm not 100% sure this is the right way to go, because semantics are different for the Map and the GridLayers. In the map you can specify a fractional maxZoom/minZoom, but maxZoom/minZoom on GridLayers will only have a meaning on integer values.

I'll make the change; we can think about semantics (and fractional GridLayer zoom levels) later.